generators/app/index.js

```diff
@@ -1,25 +1,27 @@
 'use strict';
-var generators = require('yeoman-generator');
-var isString = require('lodash.isstring');
+var Generator = require("yeoman-generator");
 
-module.exports = generators.Base.extend({
-  constructor: function () {
-    generators.Base.apply(this, arguments);
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+    
     this.option('commit', { type: String, required: false, alias: 'c',
```

Comment:
should be required then
```suggestion
    this.option('commit', { type: String, required: true, alias: 'c',
```